pycaret/time_series/forecasting/functional.py

```diff
@@ -511,17 +520,60 @@ def compare_models(
         turbo=turbo,
         errors=errors,
         fit_kwargs=fit_kwargs,
+        engines=engines,
         verbose=verbose,
     )
 
 
+@check_if_global_is_not_none(globals(), _CURRENT_EXPERIMENT_DECORATOR_DICT)
+def get_allowed_engines(estimator: str) -> Optional[str]:
```

Comment:
We should move it to tabular experiment, and implement default engines for other modules too. No need to block RC on that though!